src/renderers/webgl-fallback/WebGLBackend.js

```diff
@@ -593,8 +591,13 @@ class WebGLBackend extends Backend {
 
 				}
 
-			}
+			} else if ( resolveDepthBuffer === false ) {
+
+				const fb = renderTargetContextData.framebuffers[ renderContext.getCacheKey() ];
+				state.bindFramebuffer( gl.DRAW_FRAMEBUFFER, fb );
+				gl.invalidateFramebuffer( gl.DRAW_FRAMEBUFFER, renderTargetContextData.depthInvalidationArray );
```

Comment:
this discards the depth buffer before we do the tonemapping steps. With this, the depth is flushed to main memory which adds overhead